### ISSUE TITLE
Chef DK now includes Delivery CLI

### ISSUE THREAD

**Issue by tpetchel** [chef-cft/wombat#123]
We should install 0.15.15. It includes the CLI.

But there's a bug - it's not in the Packer build:
https://github.com/chef-cft/wombat/blob/master/packer/workstation.json#L50-L56

It should instead have something like this:
https://github.com/chef-cft/wombat/blob/master/packer/chef-server.json#L73-L74

And then we can remove this: 
https://github.com/chef-cft/wombat/blob/master/cookbooks/workstation/recipes/chef.rb#L10-L14

In summary:
1. Fix Packer build to respect `chefdk` version as specified in wombat.yml.
2. Make the default Chef DK version 0.15.15.
3. Update wombat.example.yml to specify 0.15.15, for completeness.
4. Remove explicit delivery-cli install.

**Comment by tpetchel** [comment]
Fixed.
